pkg/rt/lang.go

```diff
@@ -95,7 +95,7 @@ func installLangNS() {
 			return vm.NIL
 		}
 		for i := range vs[1:] {
-			if vs[0] != vs[i] {
+			if vs[0] != vs[i + 1] {
```

Comment:
Is this the go way? I'm a total go noob lol

Reply:
I think this should probably be a normal `for i:=1; i < len ; i++` loop. See here: https://play.golang.org/p/hiXkN537a-v

Reply:
I see...interesting.
sounds kinda silly to have "slices" and `range slice` but treat it as a normal array/list - one more go weirdness I suppose.